src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -706,6 +715,7 @@ void GcsServer::InstallEventListeners() {
     gcs_resource_manager_->OnNodeAdd(*node);
     gcs_placement_group_manager_->OnNodeAdd(node_id);
     gcs_actor_manager_->SchedulePendingActors();
+    gcs_autoscaler_state_manager_->OnNodeAdd(node_id);
```

Comment:
We should pass in `*node` and and initialize the node data inside gcs_autoscaler_state_manager.cc

Reply:
good catch. I think otherwise we will report running node with empty resources before ray syncer sends data.